step/src/main/java/com/google/sps/DatabaseHandler.java

```diff
@@ -17,12 +17,14 @@
 import java.util.Collection;
 import java.util.Map;
 import java.util.HashMap;
+import java.util.LinkedList;
 import java.util.NoSuchElementException;
 
 public final class DatabaseHandler {
   
   // TODO: replace with Datastore
-  private static Map<Long, User> idMap = new HashMap<>();
+  private static Map<Long, User> userMap = new HashMap<>();
+  private static Map<Long, Collection<Notification>> notificationMap = new HashMap<>();
```

Comment:
This is a design decision but I think it would be better to store Collection<Notification> in the User object.

Reply:
+1

Reply:
What I was trying to do here was to move more toward the model that Brenda suggested where all that's really stored in a user class is their ID and then we use that ID to look up information for the user in our database.

Reply:
Hmm, that's true. Maybe what we could do is have the User object behave like a wrapper class? Where you can do something like user.getNotifications(); and then the getNotifications method will check the database for the caller? Seems like a cleaner approach to me.

Reply:
Yes, that's the plan. Implementing that for my next PR :)

Reply:
Oh yeah, I forgot this is a temporary approach. Approved!